### ISSUE TITLE
Aux folder & file breaks cloning/extracting of code in Windows.

### ISSUE THREAD

**Issue by PhoeniXAbhisheK** [ShivangiKakkar/Burger-Builder-App#3]
Aux is a reserved keyword in Windows system and while cloning the repo, the process breaks in between and only partial files are cloned to local system.
Similarly, if downloaded and extracted the extraction process breaks midway due to the same reason.

Folder location:
Burger-Builder-App/src/hoc/Aux

System info : 
Windows 10 Pro 64-bit Operating Sytem, x64-based processor

**Comment by hackrx** [comment]
Yes, it will definitely break the cloning process, what you can do is just fork this repo and rename the aux file with another name.
And in vs code after cloning, when you run npm start it console will give you the paths where does it actually broked, so you can just change the import statement like this
import Aux from "./path/<renamedFileName>"
Hope this will help you
